diff --git a/python/pip_install/pip_repository.bzl b/python/pip_install/pip_repository.bzl
index b2cb0a8..6362882 100644
--- a/python/pip_install/pip_repository.bzl
+++ b/python/pip_install/pip_repository.bzl
@@ -13,7 +13,7 @@ def _pip_repository_impl(rctx):
         if not python_interpreter:
             fail("python interpreter not found")
 
-    rctx.file("BUILD", "")
+    rctx.file("BUILD", "exports_files(['requirements.bzl'])")
 
     # Get the root directory of these rules
     rules_root = rctx.path(Label("//:BUILD")).dirname

