---
 archive.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/archive.py b/archive.py
index 2a06dcf..ce8d9f6 100644
--- a/archive.py
+++ b/archive.py
@@ -143,7 +143,7 @@ class TarFileWriter(object):
       # Instead, we manually re-implement gzopen from tarfile.py and set mtime.
       self.fileobj = gzip.GzipFile(
           filename=name, mode='w', compresslevel=9, mtime=self.default_mtime)
-    self.tar = tarfile.open(name=name, mode=mode, fileobj=self.fileobj)
+    self.tar = tarfile.open(name=name, mode=mode, fileobj=self.fileobj, format=tarfile.USTAR_FORMAT)
     self.members = set([])
     self.directories = set([])
 
-- 
2.27.0

